script/Base.s.sol

```diff
@@ -7,7 +7,7 @@ import { Strings } from "@openzeppelin/contracts/utils/Strings.sol";
 import { console2 } from "forge-std/src/console2.sol";
 import { Script } from "forge-std/src/Script.sol";
 
-abstract contract BaseScript is Script {
+contract BaseScript is Script {
```

Comment:
Why is this not an abstract contract anymore?

Reply:
you can't initialize an abstract contract: 

https://github.com/sablier-labs/v2-core/blob/c6fcc322ff237a88302b38ec31a9b812d6e73277/test/utils/BaseScript.t.sol#L12

Reply:
Can we please make a mock for `BaseScript` in the testts instead of marking it as non-abstract?

We should not let users accidentally interact with it.

Reply:
Ok, makes sense, let's keep it as non-abstract.